core/Enclave/enclave_hsm.cpp

```diff
@@ -458,27 +496,32 @@ sgx_status_t enclave_rsa_encrypt(const uint8_t *cmk_blob,
                                  uint8_t keyspec,
                                  uint8_t padding_mode)
 {
```

Comment:
parameters not checked

Reply:
check padding mode and keyspec on line 545